EU4toV2/Data_Files/blankMod/output/decisions/converterUnions.txt

```diff
@@ -26,6 +26,29 @@ political_decisions = {
 			factor = 1
 		}
 	}
+	fixing_austria = { #Converter forgets that "Austria" and "Austria" are the same thing...
```

Comment:
What problem is this solving? Is it necessary?

Reply:
In my conversions, Austria country isn't getting its AUS tag, I'll deactivate it here because I don't think this issue affects the converter